src/widgets/AppWindow.js

```diff
@@ -1948,10 +1978,10 @@ export default class AppWindow extends createjs.Container {
 
     for (let i = 0; i < sliderData.length; i++) {
       const name = sliderData[i].label;
-      const min = sliderData[i].min;
-      const max = sliderData[i].max;
-      const step = sliderData[i].step;
-      const value = sliderData[i].value;
+      const { min } = sliderData[i];
+      const { max } = sliderData[i];
+      const { step } = sliderData[i];
+      const { value } = sliderData[i];
```

Comment:
Nit: these lines can be combined into `const { min, max, step, value } = sliderData[i];`

Reply:
updated